unicornherder/command.py

```diff
@@ -24,6 +24,9 @@
                     help='Path to the pidfile that unicorn will write')
 parser.add_argument('-t', '--timeout', default=30, type=int, metavar='30', dest='boot_timeout',
                     help='Timeout in seconds to start workers')
+parser.add_argument('-o', '--overlap', default=120, type=int, metavar='120',
+                    dest='overlap',
+                    help='Time to wait before killing old unicorns when reloading')
```

Comment:
`boot_wait` and `boot_time` are probably similar enough to `boot_timeout` to confuse people; we picked `overlap` as it seemed to express the intent of the flag (that is, how long the old and new processes should both be active before we tell the old process to start tidying up its workers). We also considered something along the lines of `grace_period`, but figured that might be too similar to Gunicorn’s [graceful timeout](http://docs.gunicorn.org/en/develop/configure.html#graceful-timeout).

Would `overlap` be alright if it came with more expressive help text?